### DISCUSSION TITLE
Do we have ETCD best practices?

### DISCUSSION THREAD

**Discussion by Lewisyixin** [apache/apisix#6872] [Q&A]
准备上生产环境了，但是不太清楚所需要的etcd集群的配置多少比较合适，担心如果给少了拖累集群。
但是按照我的理解，正常访问请求不会读取etcd的数据，因此网关的qps和etcd配置没有直接联系，如果只是定期更新内存数据的话，我理解应该不需要太高的配置。
如果有哪些同伴已经在生产环境运行一段时间的话 能够分享一下etcd的耗用资源的话，感激不尽~

**Comment by tokers** [comment]
See https://etcd.io/docs/v3.3/op-guide/hardware/.
